contrib/opentimelineio_contrib/adapters/advanced_authoring_format.py

```diff
@@ -288,14 +339,15 @@ def _transcribe(item, parents, edit_rate, indent=0):
             _encoded_name(item)), indent)
         result = otio.schema.Clip()
 
-        # Evidently the last mob is the one with the timecode
-        mobs = _find_timecode_mobs(item)
-
-        # Get the Timecode start and length values
-        last_mob = mobs[-1] if mobs else None
-        timecode_info = _extract_timecode_info(last_mob) if last_mob else None
+        # Get all relevant mobs down the tree and their source clips
+        # These are necessary for calculating correct starting values
+        mc_tc_info = _find_mob_chain_and_timecode(item, edit_rate)
+        mobs, timecode_info, source_start_multiplier = mc_tc_info
 
         source_start = int(metadata.get("StartTime", "0"))
+        if source_start_multiplier:
+            source_start = int(source_start * source_start_multiplier)
```

Comment:
Is the multiplier just from varying frame rate clips, or is this from timewarp effects in the composition?
0 or None means it was not found, and should be ignored.